### PR TITLE
Feature: Replaced string "Layout mode" with "Layout" for simplicity

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

As the title.

## PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   _None_

## Screenshots

_None_

### CODE REVIEW

File: src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs [hunk 1]
```diff
@@ -99,7 +99,7 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 			{
 				new ContextMenuFlyoutItemViewModel()
 				{
-					Text = "LayoutMode".GetLocalizedResource(),
```
Comment (yaira2) [inline src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs:102]: Can you do the same for the settings dialog and wherever else this string is used?

Comment (0x5bfa) [reply src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs:102]: Do you like to remove this resource?

Comment (yaira2) [reply src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs:102]: That would make sense

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Data/Factories/ContentPageContextFlyoutFactory.cs:102]: Thanks!

**Pull request merged**